app/scripts/background.js

```diff
@@ -892,6 +891,9 @@ export function setupController(
       controller.isClientOpen = true;
       controller.setupTrustedCommunication(portStream, remotePort.sender);
 
+      // initialize the request to fetch remote feature flags
```

Comment:
I think we should wrap this in a try catch, we can just use `log.error` in the catch

Reply:
addressed in 5204e9c4ff436e00a8f512319bcf0c23b88d6393